Riot/Modules/SideMenu/SideMenuCoordinator.swift

```diff
@@ -423,12 +444,46 @@ extension SideMenuCoordinator: SpaceMembersCoordinatorDelegate {
     }
 }
 
+// MARK: - CreateRoomCoordinatorDelegate
+extension SideMenuCoordinator: CreateRoomCoordinatorDelegate {
+    func createRoomCoordinator(_ coordinator: CreateRoomCoordinatorType, didCreateNewRoom room: MXRoom) {
+        coordinator.toPresentable().dismiss(animated: true) {
```

Comment:
`SideMenuCoordinator` is a bit different and we don't add child coordinator. Nevertheless the instance of `CreateRoomCoordinatorDelegate` is kept in the ivar `createRoomCoordinator` which is set to `nil`.

Reply:
Ah, I saw `add(childCoordinator: createRoomCoordinator)` [here](https://github.com/vector-im/element-ios/blob/4a0f86ee8ebc7cfa62c717b6f5b2a59cc8c46ead/Riot/Modules/SideMenu/SideMenuCoordinator.swift#L300) and just assumed it would need to be removed.